dashbuilder/dashbuilder-backend/dashbuilder-kie-server-backend/src/main/java/org/dashbuilder/kieserver/backend/rest/KieServerQueryClient.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dashbuilder.kieserver.backend.rest;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+
+import org.dashbuilder.kieserver.KieServerConnectionInfo;
+
+/**
+ * Run queries on Kie Server using its REST API
+ * 
+ */
+@ApplicationScoped
+public class KieServerQueryClient {
+
+    private static final String MEDIA_TYPE = MediaType.APPLICATION_JSON;
```

Comment:
I did this so I can easiy switch between JSON and XML in all methods. If I don't this I may have to change to XML, for example, and mistakely request JSON and send XML bodies (and vice versa). To avoid confusion I renamed my constant to REQUEST_MEDIA_TYPE.